numpy/core/_add_newdocs.py

```diff
@@ -7970,62 +7972,164 @@ def luf(lamdaexpr, *args, **kwargs):
 #
 ##############################################################################
 
-add_newdoc('numpy.core.numerictypes', 'bool_',
-    """NumPy's Boolean type.  Character code: ``?``.  Alias: bool8""")
+def numeric_type_aliases(aliases):
+    def type_aliases_gen():
+        for (alias, doc) in aliases:
+            alias_type = getattr(_numerictypes, alias, None)
+            if alias_type is not None:
+                yield (alias_type, alias, doc)
```

Comment:
Strictly speaking this would be better as:
```
try:
    alias_type = getattr(_numerictypes, alias)
except AttributeError:
    pass
else:
    yield (alias_type, alias, doc)
```